grammars/css.cson

```diff
@@ -498,7 +498,13 @@
         'captures':
           '1':
             'name': 'punctuation.definition.entity.css'
-        'match': '(\\.)[a-zA-Z0-9_-]+'
+        'match': '''(?x)
+          (\\.)
+          -?+                   # possessive to avoid matching '-' followed by <0-9>
+          [\\w-&&[^0-9]]        # Do NOT change 0-9 to \d.
```

Comment:
Coz they're not the same, except when using ASCII encoding. Just like `\w` matches much more than `[0-9_A-Za-z]` (see the new specs), `\d` matches much more than `[0-9]`: https://github.com/kkos/oniguruma/blob/master/src/unicode_property_data.c#L687-L739

`&&` in character class is used to obtain an intersection of two sets, `[\w-]` and `[^0-9]` in this case, so it's effectively rejecting `[0-9]` from the set `[\w-]`.